components/api_version/src/api_v2.rs

```diff
@@ -151,16 +153,27 @@ impl APIVersion for APIV2 {
     }
 }
 
+impl APIV2 {
+    pub fn append_ts_on_encoded_bytes(encoded_bytes: &mut Vec<u8>, ts: TimeStamp) {
```

Comment:
why not to use `Key::append_ts`?

Reply:
`Key::append_ts` need moving a `Vec<u8>` in and out, and is not convenient to use. E.g., if using `Key::append_ts`, codes with `append_ts_on_encoded_bytes ` would be as following:
```
req.mut_put().set_key(APIV2::append_ts_on_encoded_bytes(req.mut_put().take_key(), ts))
```